mods/cnc/chrome/ingame.yaml

```diff
@@ -106,6 +106,11 @@ Container@OBSERVER_WIDGETS:
 					Y: 1
 					Width: PARENT_RIGHT-2
 					Height: PARENT_BOTTOM-2
+				VqaPlayer@PLAYER:
```

Comment:
Whoops, you added this to the observer's radar.  The player's radar bin is a bit further down.